transport-classes-epoll/src/main/java/io/netty/channel/epoll/NativeDatagramPacketArray.java

```diff
@@ -158,7 +159,8 @@ private static InetSocketAddress newAddress(byte[] addr, int addrLen, int port,
      * Used to pass needed data to JNI.
      */
     @SuppressWarnings("unused")
-    final class NativeDatagramPacket {
+    @UnstableApi
+    public final class NativeDatagramPacket {
```

Comment:
how is this exposed as public ?

Reply:
The `io.netty.channel.epoll.Native` class is `public` and has a `static` method `sendmmsg` that is `public` and take a `NativeDatagramPacket` as an argument.